flink-runtime/src/main/java/org/apache/flink/runtime/executiongraph/AccessExecutionGraph.java

```diff
@@ -70,6 +70,8 @@
 	 */
 	String getFailureCauseAsString();
 
+	long getFailureTimeStamp();
```

Comment:
`TimeStamp` with upper case `S` is inconsistent with other uses and I think incorrect. Could you replace it with `timestamp` here in other classes?